base_classes/NXbeam.nxdl.xml

```diff
@@ -61,11 +61,45 @@
         <doc>Distance from sample. Note, it is recommended to use NXtransformations instead.</doc>
     </field>
     <field name="incident_energy" type="NX_FLOAT" units="NX_ENERGY">
-        <doc>Energy carried by each particle of the beam on entering the beamline component</doc>
+        <doc>
+            Energy carried by each particle of the beam on entering the beamline component.
+            
+            In the case of a monochromatic beam this is the scalar energy.
+            Several other use cases are permitted, depending on the
+            presence of other incident_energy_X fields.
```

Comment:
Add backticks to emphasize field names to RST formatting

Reply:
changed, thanks